.buildkite/pipelines/es_snapshots/promote.yml

```diff
@@ -12,3 +12,7 @@ steps:
     command: .buildkite/scripts/steps/es_snapshots/promote.sh
     agents:
       queue: kibana-default
+  - wait
+  - trigger: kibana-agent-packer-cache
```

Comment:
I assume we'll keep the current daily trigger but just trigger a second packer job if ES snapshots are promoted too? Does that job cancel in progress jobs if a second one is triggered? Might have a race condition to deal with if not.

Reply:
Yep, that's the idea. I just added `cancel_intermediate_builds` to the cache pipeline.